nbdev/clean.py

```diff
@@ -87,24 +95,31 @@ def process_write(warn_msg, proc_nb, f_in, f_out=None, disp=False):
         warn(f'{warn_msg}')
         warn(e)
 
-# %% ../nbs/11_clean.ipynb 14
+# %% ../nbs/11_clean.ipynb 15
 @call_parse
 def nbdev_clean(
     fname:str=None, # A notebook name or glob to clean
     clear_all:bool=False, # Clean all metadata and outputs
     disp:bool=False,  # Print the cleaned outputs
     stdin:bool=False # Read notebook from input stream
 ):
-    "Clean all notebooks in `fname` to avoid merge conflicts"
+    """Clean all notebooks in `fname` to avoid merge conflicts.
```

Comment:
We use single quoted docstrings (i.e 1-line) -- use markdown in the notebook for extra info for users.